### PR TITLE
Deployment info/ URL set up

### PR DESCRIPTION
**Pull request by donuty**

### CODE REVIEW

File: client/src/components/relatedItems/RelatedItemsWidget.jsx [hunk 1]
```diff
@@ -37,25 +43,72 @@ class RelatedItemsWidget extends Component {
...
 
-
   setOverviewIdData() {
-
     this.setState({
       loaded: false
     })
-
     const endpoints = [
       `http://localhost:8080/products/${this.state.overviewId}`,
```
Comment (DarianAltF4) [inline client/src/components/relatedItems/RelatedItemsWidget.jsx:55]: Once you deploy, you will need to change your endpoints to the following:
`'/products/${this.state.overviewId}'`
The reason behind this is that you would get an error on the deployed instance saying it doesnt know what localhost is. You can do this now as well. These changes will work on a deployed state and local state.

**Review by DarianAltF4 — APPROVED** [review]
Everything looks good. Just change the endpoint that I suggested to prevent deployment issues.

File: client/src/components/relatedItems/RelatedItemsWidget.jsx (hunk 1 above)
Comment (DarianAltF4) [inline client/src/components/relatedItems/RelatedItemsWidget.jsx:55]: Once you deploy, you will need to change your endpoints to the following:
'/products/${this.state.overviewId}'
The reason behind this is that you would get an error on the deployed instance saying it doesnt know what localhost is. You can do this now as well. These changes will work on a deployed state and local state.

**Review by DarianAltF4 — CHANGES_REQUESTED** [review]
Change the suggested urls so you don't have errors in deployment then I will approve the PR 😄

File: client/src/components/relatedItems/RelatedItemsWidget.jsx (hunk 1 above)
Comment (donuty) [reply client/src/components/relatedItems/RelatedItemsWidget.jsx:55]: Sounds good!  I'll do that now.

**Pull request closed without merging**